src/solver/application/application.cpp

```diff
@@ -113,71 +95,31 @@ void Application::prepare(int argc, char* argv[])
         pStudy = nullptr;
         return;
     }
+}
 
-    // Perform some checks
-    checkAndCorrectSettingsAndOptions(pSettings, options);
-
-    pSettings.checkAndSetStudyFolder(options.studyFolder);
-
-    checkStudyVersion(pSettings.studyFolder);
-
-    // Determine the log filename to use for this simulation
-    resetLogFilename();
-
+void Application::startSimulation(Data::StudyLoadOptions options)
+{
     // Starting !
-#ifdef GIT_SHA1_SHORT_STRING
+    #ifdef GIT_SHA1_SHORT_STRING
     logs.checkpoint() << "Antares Solver v" << ANTARES_VERSION_STR << " (" << GIT_SHA1_SHORT_STRING
                       << ")";
-#else
+    #else
     logs.checkpoint() << "Antares Solver v" << ANTARES_VERSION_STR;
-#endif
+    #endif
     WriteHostInfoIntoLogs();
 
-    // Write command-line options into logs
-    // Incidentally, it also seems to contain the full path to the executable
-    logs.info();
-    WriteCommandLineIntoLogs(argc, argv);
+    WriteCommandLineIntoLogs(pArgc, pArgv);
 
     logs.info() << "  :: log filename: " << logs.logfile();
-    // Temporary use a callback to count the number of errors and warnings
+
     logs.callback.connect(this, &Application::onLogMessage);
 
-    // Allocate a study
     pStudy = std::make_shared<Antares::Data::Study>(true /* for the solver */);
 
-    // Setting global variables for backward compatibility
     pParameters = &(pStudy->parameters);
 
-    // Loading the study
     readDataForTheStudy(options);
 
-    // Some more checks require the existence of pParameters, hence of a study.
-    // Their execution is delayed up to this point.
-    checkOrtoolsUsage(
-      pParameters->unitCommitment.ucMode, pParameters->ortoolsUsed, pParameters->ortoolsSolver);
-
-    checkSimplexRangeHydroPricing(pParameters->simplexOptimizationRange,
-                                  pParameters->hydroPricing.hpMode);
-
-    checkSimplexRangeUnitCommitmentMode(pParameters->simplexOptimizationRange,
-                                        pParameters->unitCommitment.ucMode);
-
-    checkSimplexRangeHydroHeuristic(pParameters->simplexOptimizationRange, pStudy->areas);
-
-    if (pParameters->adqPatchParams.enabled)
-        pParameters->adqPatchParams.checkAdqPatchParams(pParameters->mode,
-                                                        pStudy->areas,
-                                                        pParameters->include.hurdleCosts);
-
-    bool tsGenThermal
-      = (0 != (pParameters->timeSeriesToGenerate & Antares::Data::TimeSeriesType::timeSeriesThermal));
-
-    checkMinStablePower(tsGenThermal, pStudy->areas);
-
-    checkFuelCostColumnNumber(pStudy->areas);
-    checkCO2CostColumnNumber(pStudy->areas);
```

Comment:
These checks disappeared, right ? Is this intentional ?

Reply:
Totally unintentional. Probably a merge/rebase mistake.